codes/classical/q-ary_digits/generalized_reed_solomon.yml

```diff
@@ -0,0 +1,30 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: generalized_reed_solomon
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Generalized Reed-Solomon code'
+introduced: '\cite{manual:{MacWilliams, Florence Jessie, and Neil James Alexander Sloane. \emph{The theory of error correcting codes}. Ch.10, Sec.8. Vol. 16. Elsevier, 1977}'
+
+description: 'Stub.'
+#
+#protection: 'Corrects erasures on \(n-k\) coordinates.'
+
+#notes:
+#  - ''
+#
+#realizations:
+#  - 'Numerous applications: computer hard-disks, DVDs, 3G telephone networks (IS-2000, Release D), NASA deep-space exploration, etc.'
+
+relations:
+  parents:
+    - code_id: 'q-ary_linear'
+      detail: ''
+
+  #cousins:
```

Comment:
Is there any reason Hamming should be considered as a cousin? If not, please comment out.

Reply:
Done.